src/test/java/KoelTesting.java

```diff
@@ -16,7 +16,7 @@ public static void logIntoKoel(WebDriver driver, String userName, String passwor
     public static void goToKoel(WebDriver driver){
         driver.manage().window().maximize();
         driver.get("https://bbb.testpro.io/");
-        waitforSeconds(driver,2);
+        waitforSeconds(driver,3);
```

Comment:
You got this problem, please correct it. 

Check login and password

![image](https://user-images.githubusercontent.com/104761482/211012818-96d785d5-fbd0-4b46-ae1b-4e3485fb95e0.png)


![image](https://user-images.githubusercontent.com/104761482/211012761-c7a2b6d1-bc19-4351-8293-28c08b52faef.png)